projects/ngx-drag-to-select/src/lib/select-container.component.ts

```diff
@@ -125,12 +125,20 @@ export class SelectContainerComponent implements AfterViewInit, OnDestroy {
   @Output()
   itemDeselected = new EventEmitter<any>();
 
+  @Output()
+  selectionStarted = new EventEmitter<any>();
```

Comment:
We aren't really emitting a value, so `selectionStarted = new EventEmitter();` would be better here.

Reply:
Good point. This should be `void` or I could emit the type completely. Spot on Tim!